src/core_plugins/metrics/public/components/vis_types/table/series.js

```diff
@@ -0,0 +1,155 @@
+import AddDeleteButtons from '../../add_delete_buttons';
+import SeriesConfig from './config';
+import Sortable from 'react-anything-sortable';
+import Tooltip from '../../tooltip';
+import createTextHandler from '../../lib/create_text_handler';
+import createAggRowRender from '../../lib/create_agg_row_render';
+
+function TopNSeries(props) {
+  const {
+    model,
+    onAdd,
+    onChange,
+    onDelete,
+    disableDelete,
+    disableAdd,
+    selectedTab,
+    visible
+  } = props;
+
+  const handleChange = createTextHandler(onChange);
+  const aggs = model.metrics.map(createAggRowRender(props));
+
+  let caretClassName = 'fa fa-caret-down';
+  if (!visible) caretClassName = 'fa fa-caret-right';
+
+  let body = null;
+  if (visible) {
+    let metricsClassName = 'kbnTabs__tab';
+    let optionsClassname = 'kbnTabs__tab';
+    if (selectedTab === 'metrics') metricsClassName += '-active';
+    if (selectedTab === 'options') optionsClassname += '-active';
+    let seriesBody;
+    if (selectedTab === 'metrics') {
+      const handleSort = (data) => {
+        const metrics = data.map(id => model.metrics.find(m => m.id === id));
+        props.onChange({ metrics });
+      };
+      seriesBody = (
+        <div>
+          <Sortable
+            style={{ cursor: 'default' }}
+            dynamic={true}
+            direction="vertical"
+            onSort={handleSort}
+            sortHandle="vis_editor__agg_sort"
+          >
+            { aggs }
+          </Sortable>
+        </div>
+      );
+    } else {
+      seriesBody = (
+        <SeriesConfig
+          panel={props.panel}
+          fields={props.fields}
+          model={props.model}
+          onChange={props.onChange}
+        />
+      );
+    }
+    body = (
+      <div className="vis_editor__series-row">
+        <div className="kbnTabs sm">
+          <div
+            className={metricsClassName}
+            onClick={() => props.switchTab('metrics')}
+          >
+            Metrics
+          </div>
+          <div
+            className={optionsClassname}
+            onClick={() => props.switchTab('options')}
+          >
+            Options
+          </div>
+        </div>
+        {seriesBody}
+      </div>
+    );
+  }
+
+  let dragHandle;
+  if (!props.disableDelete) {
+    dragHandle = (
+      <Tooltip text="Sort">
+        <div className="vis_editor__sort thor__button-outlined-default sm">
+          <i className="fa fa-sort" />
+        </div>
+      </Tooltip>
+    );
+  }
+
+  return (
+    <div
+      className={`${props.className} vis_editor__series`}
+      style={props.style}
+      onMouseDown={props.onMouseDown}
+      onTouchStart={props.onTouchStart}
+    >
+      <div className="vis_editor__container">
+        <div className="vis_editor__series-details">
+          <div onClick={props.toggleVisible}><i className={caretClassName}/></div>
+          <div className="vis_editor__row vis_editor__row_item">
+            <input
```

Comment:
This input is missing a [correct label](https://github.com/elastic/kibana/blob/master/style_guides/accessibility_guide.md#label-every-form-element).